src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/PaymentAccountValidator.java

```diff
@@ -39,24 +41,25 @@ public void validatePaymentAccounts(Set<String> paymentAccounts, String orgId) {
 
     public static void checkPbaNumberIsValid(Set<String> paymentAccounts) {
         String invalidPbas = paymentAccounts.stream()
-                .filter(pbaAccount -> {
-                    if (!StringUtils.isBlank(pbaAccount) && pbaAccount.length() == 10) {
-                        Pattern pattern = Pattern.compile("(?i)pba\\w{7}$");
-                        Matcher matcher = pattern.matcher(pbaAccount);
-                        if (matcher.matches()) {
-                            return false;
-                        }
-                    }
-                    return true;
-                })
+                .filter(PaymentAccountValidator::isPbaInvalid)
                 .collect(Collectors.joining(", "));
 
         if (!StringUtils.isEmpty(invalidPbas)) {
-            throw new InvalidRequest("PBA numbers must start with PBA/pba and be followed by 7 alphanumeric "
-                    .concat("characters. The following PBAs entered are invalid: " + invalidPbas));
+            throw new InvalidRequest(ERROR_MSG_PBA_INVALID_FORMAT
+                    .concat(". The following PBAs entered are invalid: " + invalidPbas));
```

Comment:
"The following PBAs entered are invalid" can also be made as constant

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/610783f34efa46b14714ed71602afd4abced4741